src/components/WeeklySummariesReport/WeeklySummariesReport.jsx

```diff
@@ -17,18 +17,27 @@ export class WeeklySummariesReport extends Component {
     error: null,
     loading: true,
     summaries: [],
-    activeTab: '1',
+    activeTab: '2'
   };
 
   async componentDidMount() {
     await this.props.getWeeklySummariesReport();
+    window.addEventListener("popstate", () => {
```

Comment:
Could you explain the reasoning behind using this event listener? My guess is that it's supposed to cleanup localStorage after navigating to other pages (The state wouldn't be needed on those other pages) but could be wrong.

Reply:
Attaching a link to pop state event listener from mdn.

https://developer.mozilla.org/en-US/docs/Web/API/Window/popstate_event

This specifically tracks browser events such as pressing the back button, or forward. Even gives you history (if you need it). So we “listen” to events where the user would press the back button on the browser. 

Yes correct, we don’t need it on other pages. And this PR should be ensuring clearing local storage.

Reply:
It seems that the `componentWillUnmount` runs when pressing the forward/back button, so I'm wondering if it's even necessary to use an event listener at all. What are your thoughts?

Reply:
Pushed a commit that reflects the change. Replied to your comment below as I didn't see this comment on this thread earlier.